### ISSUE TITLE
Outdated Oracle JDK9 trust store on Mac

### ISSUE THREAD

**Issue by lbulej** [renaissance-benchmarks/renaissance#249]
Recent PRs have trouble passing the Travis bundle stage with Oracle JDK9 on MacOS X.

The problem looks like an outdated trust store:
- https://ao.gl/solved-pkix-path-building-failed-sun-security-provider-certpath-suncertpathbuilderexception-unable-to-find-valid-certification-path-to-requested-target/
- http://littlebigextra.com/how-to-fix-pkix-path-building-failed-sun-security-provider-certpath-suncertpathbuilderexception/
- https://confluence.atlassian.com/kb/unable-to-connect-to-ssl-services-due-to-pkix-path-building-failed-error-779355358.html

We can try to include the missing certificates in the repository and install them during Travis build:
```shell
find ./certs -name '*.pem' | while read CERT; do
    keytool -importcert -trustcacerts -keystore "$JAVA_HOME/jre/lib/security/cacerts" -storepass changeit -alias $(basename "$CERT" .pem) -file "$CERT" -noprompt
done
```

However, I'm not sure if we have the permissions to modify that file.

Alternatively, we would need to copy the `cacerts` file from the JDK system directory, modify it, and then use the `-Djavax.net.ssl.trustStore` property to point to the modified `cacerts` file. This might be amenable to integration in the `sbt` launch script and startup configuration (or creating a .sbt file with the JVM option), because `sbt` is the affected party here.

**Comment by lbulej** [comment]
Another alternative would be to play with `osx_image`. We are currently using `xcode9.3`, but `xcode9.4`  is the default:
- https://docs.travis-ci.com/user/reference/osx

**Comment by lbulej** [comment]
> Bumping `osx_image` to `xcode9.4` appears to be helping.

OK, not really, but running with `oraclejdk8` instead of `oraclejdk9` seems to.

**Comment by farquet** [comment]
How about using OracleJDK11 instead of OracleJDK9?

I think sticking the Travis gate to LTS versions of the JDK would be better and easier to maintain. So 8, 11 and 17 when the time comes.

**Comment by lbulej** [comment]
We did not have a clear policy on the choice of JDKs to test on and I don't know why we chose JDK9 on OS X.  Also, it's not clear why JDK9 should have an outdated trust store compared to JDK8, but we can work with that now.  Ideally, we would want to test more JDKs, but it's becoming unwieldy. 

This issue actually forced me to disassemble and reassemble the Travis CI setup over the weekend in the `rework-ci-new` branch and it appears to work for JDK8, JDK9, and JDK11 (including OpenJ9). I'm currently cleaning it up in a different branch for merge (the `rework-ci-new` branch just captures my experiments).

That said, it will not remove the need to choose a reasonable set of JDKs for Travis, because the wait is just too long.
